tests/functional/aws-node-sdk/test/bucket/putBucketNotification.js

```diff
@@ -89,6 +89,17 @@ describe('aws-sdk test put notification configuration', () => {
                 });
             });
 
+        it('should put empty notification configuration', done => {
+            const params = {
+                Bucket: bucket,
+                NotificationConfiguration: {},
+            };
+            s3.putBucketNotificationConfiguration(params, err => {
+                assert.ifError(err);
+                done();
+            });
```

Comment:
Mocha will handle the error in the done callback so you can just pass that straight in.
```suggestion
            s3.putBucketNotificationConfiguration(params, done);
```